cmd/helper/main.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	"github.com/kubeshop/kusk-gateway/internal/helper/httpserver"
+	"github.com/kubeshop/kusk-gateway/internal/helper/management"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/keepalive"
+)
+
+// Making these global
+
+func main() {
+
+	var (
+		helperConfigurationManagerServiceAddress string
+		log                                      *zap.SugaredLogger
+		fleetID                                  string
+		nodeName                                 string
```

Comment:
This isn't passed as a flag so it can be declared on line 37 as

`nodeName, err := os.Hostname()`